### ISSUE TITLE
Icons in `new` roles view not hidden from screen readers

### ISSUE THREAD

**Issue by metamoni** [joemasilotti/railsdevs.com#551]
## Describe the bug
The `briefcase` and `user_group` SVGs are being picked up by VoiceOver but have no accessible titles, so screen readers announce "image" with no other context.

Seeing as these images and decorative and not functional, they should be hidden from screen readers. 

## To reproduce

1. Go to [https://railsdevs.com/role/new](https://railsdevs.com/role/new)
2. Turn VoiceOver on: `command + F5`
3. Press `control + option + Right Arrow` to read the page content
4. Notice that when going over the `briefcase` and `user_group` icons VoiceOver announces "image" with no additional context
5. Turn VoiceOver off: `command + F5`

## Expected behavior
VoiceOver should skip icons entirely

## Actual behavior
VoiceOver announces "image"

## Screenshots
https://user-images.githubusercontent.com/22390758/178141929-707a7ee0-c04b-43e7-a86b-8d19788ca461.mov

## Additional context
Helpful resources:
- [Accessibility in `inline_svg`](https://github.com/jamesmartin/inline_svg#accessibility)
- [More on accessible SVGs](http://www.a11yonrails.com/images#accessible-svgs)

**Comment by crabbits** [comment]
@joemasilotti I'm happy to work on this issue if you wish

**Comment by joemasilotti** [comment]
Sure thing – all yours!

**Comment by crabbits** [comment]
@joemasilotti @metamoni opened a PR here: https://github.com/joemasilotti/railsdevs.com/pull/562, looking for to feedback :)

**Comment by joemasilotti** [comment]
Closed by #562.
